src/main/scala/io/sdkman/changelogs/SparkMigrations.scala

```diff
@@ -233,4 +233,18 @@ class SparkMigrations {
       .insert()
       .asCandidateDefault()
   }
+
+  @ChangeSet(
+    order = "018",
+    id = "018-add_spark_3.0.3",
+    author = "ChethanUK"
+  )
+  def migration018(implicit db: MongoDatabase) = {
+    Version(
+      "spark",
+      "3.1.1",
+      "https://archive.apache.org/dist/spark/spark-3.0.3/spark-3.0.3-bin-hadoop3.2.tgz"
```

Comment:
It seems that this is for Spark 3.0.3, not 3.1.1. It has already been merged but I'll back this one out.